File: problems/0004_median-of-two-sorted-arrays/concat-sort/solution.js
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @return {number}
 */
var findMedianSortedArrays = function (nums1, nums2) {
    'use strict'
    const nums = nums1.concat(...nums2).sort((l, r) => l - r),
          i = parseInt(nums.length / 2)
    return (nums.length % 2 === 0 ? (nums[i - 1] + nums[i]) / 2 : nums[i])
};

File: problems/0004_median-of-two-sorted-arrays/concat-sort/Solution.java
import java.util.Arrays;

class Solution {
    public double findMedianSortedArrays(int[] nums1, int[] nums2) {
        final int[] nums = new int[nums1.length + nums2.length];
        System.arraycopy(nums1, 0, nums, 0, nums1.length);
        System.arraycopy(nums2, 0, nums, nums1.length, nums2.length);
        Arrays.sort(nums);
        final int i = nums.length / 2;
        return (nums.length % 2 == 0 ? (nums[i - 1] + nums[i]) / 2.0 : nums[i]);
    }
}
